test/blackbox-tests/test-cases/cram/cram-action-shell-opt.t

```diff
@@ -0,0 +1,158 @@
+Check that cram tests run without shell option specified
+
+  $ echo '(lang dune 3.10)' > dune-project
+
+  $ cat > dune <<EOF
+  > (rule
+  >   (target foo.cram.corrected)
+  >   (mode (promote (until-clean)))
+  >   (action (cram foo.cram))
+  > )
+  > EOF
```

Comment:
```suggestion
  > (rule
  >  (target foo.cram.corrected)
  >  (mode (promote (until-clean)))
  >  (action (cram foo.cram)))
  > EOF
```

Reply:
And for the rest of the stanzas. Usually we stick to a single space indent and leave no brackets on their own lines.

Reply:
addressed in b7777bd90